### ISSUE TITLE
Medium trace concentration in boundaries

### ISSUE THREAD

**Issue by rubenbaetens** [open-ideas/IDEAS#547]
The default value for `C` in `IDEAS.Fluid.Sources.Boundary_pT` is now `fill(0,Medium.nC)`, shouldn't this be `Medium.C_nominal` ? 

In fact i struggle correctly modelling CO2 in the zone. It seems the infiltration air has zero traces of CO2, thus dropping below 400 ppm in some spaces, while it uses `Medium.C_nominal` and i define my medium as follows:
 ```
package AirCO2 "MoistAirUnsaturated with CO2"
  extends IDEAS.Media.Air(
  extraPropertiesNames=fill("CO2"), nC=1, final C_nominal={0.000288});
end AirCO2;
 ```

**Comment by rubenbaetens** [comment]
Additionally, given the above media definition, i expected the starting conditions of the lumpedVolume to have a `C` equal to `C_nominal` but this seems to be zero.

**Comment by rubenbaetens** [comment]
@Mathadon Could you maybe add an example in `IDEAS.Buildings.Examples.InternalGainExample` including CO2 ?

**Comment by Mathadon** [comment]
You may indeed argue that setting `Medium.C_nominal` makes more sense. 
Will try to get you an example tomorrow.

**Comment by Mathadon** [comment]
See InternalGainExample.mo in https://github.com/open-ideas/IDEAS/tree/issue547_example 
still need to run unit tests

**Comment by Mathadon** [comment]
Actually, it seems that `C_nominal` does not exist. You can manually set `C_start` however.

**Comment by rubenbaetens** [comment]
What do you mean with `C_nominal` does not exist ?

**Comment by Mathadon** [comment]
That parameter does not exist in the Medium package. So there is not a proper way to define the nominal C concentration.

**Comment by rubenbaetens** [comment]
In that case the information in the Annex60 code should be updated, cause the `package content` description of `IDEAS.Media.Air` states `C_nominal=1.0e-6*ones(nC)`

**Comment by Mathadon** [comment]
In that case the LumpedVolumeDeclaration model should contain:

```
  parameter Medium.ExtraProperty C_start[Medium.nC](
       quantity=Medium.extraPropertiesNames)=Medium.C_nominal
    "Start value of trace substances"
    annotation (Dialog(tab="Initialization", enable=Medium.nC > 0));
  parameter Medium.ExtraProperty C_nominal[Medium.nC](
       quantity=Medium.extraPropertiesNames) = Medium.C_nominal
    "Nominal value of trace substances. (Set to typical order of magnitude.)"
   annotation (Dialog(tab="Initialization", enable=Medium.nC > 0));
```
Can you make a pull request for this in Annex 60?

**Comment by Mathadon** [comment]
If you then also use

```
package MediumAir = IDEAS.Media.Air(extraPropertiesNames={"CO2"}, C_nominal={400*44/29/1e6});
```

or

```
package MediumAir = IDEAS.Media.Air(extraPropertiesNames={"CO2"}, C_nominal={ppmCO2*44/29/1e6});
```

then it works.

**Comment by rubenbaetens** [comment]
By just adding the parameters in the LumpedVolumeDeclaration, no equation or connection needed ?

**Comment by rubenbaetens** [comment]
Addressed in #548.
